ui/app/src/components/Footer.tsx

```diff
@@ -46,9 +46,8 @@ export default function Footer(): ReactElement {
           </a>
         </li>
         <li>
-          {isLoading ? (
-            <CircularProgress size="1rem" />
-          ) : data !== undefined && data.version !== '' ? (
+          {isLoading && <CircularProgress size="1rem" />}
+          {!isLoading && data?.version && (
```

Comment:
Same as below, we should extract this into a function as removing the nested ternary does not make it more readable or maintanable